spec/helpers/application_helper_spec.rb

```diff
@@ -20,4 +20,12 @@
       end
     end
   end
+
+  describe "#markdown" do
+    context "converts to markdown" do
+      it "when given valid markdown" do
+        expect(helper.markdown("_foo_")).to eq("<p><em>foo</em></p>\n")
+      end
```

Comment:
If you want to tidy you can use `match` instead of `eq` to check that something contains something without doing the whole thing. `match` accepts a string or regex.
For instance:
```
expect(helper.markdown("_foo_")).to match("<em>")
```